test/bitcoin_functional/functional/feature_notifications.py

```diff
@@ -66,23 +66,5 @@ def run_test(self):
             txids_rpc = list(map(lambda t: t['txid'], self.nodes[1].listtransactions("*", block_count)))
             assert_equal(sorted(txids_rpc), sorted(os.listdir(self.walletnotify_dir)))
 
-        # Mine another 41 up-version blocks. -alertnotify should trigger on the 51st.
```

Comment:
This is not from upstream, right? You just removing these because they are annoying and pointless? +1 then :D

Reply:
No this test case was removed in the elements side of tests. That feature
is gone.